File: 2° Periodo/Prog 2  JAVA C++/POO_TRABALHO_1/src/Pessoa.java
public class Pessoa {
    private String nome, endereco, cpf;
    private char sexo;
    private int telefone, identidade;

    public Pessoa() {
    }

    public Pessoa(String nome, String endereco, String cpf, char sexo, int telefone, int identidade) {
        this.nome = nome;
        this.endereco = endereco;
        this.cpf = cpf;
        this.sexo = sexo;
        this.telefone = telefone;
        this.identidade = identidade;
    }

    public void imprimir() {
        System.out.println("Nome: " + nome);
        System.out.println("Sexo: " + sexo);
        System.out.println("CPF: " + cpf);
        System.out.println("Endereco: " + endereco);
        System.out.println("Telefone: " + telefone);
        System.out.println("Identidade" + identidade);
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEndereco() {
        return endereco;
    }

    public void setEndereco(String endereco) {
        this.endereco = endereco;
    }

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public char getSexo() {
        return sexo;
    }

    public void setSexo(char sexo) {
        this.sexo = sexo;
    }

    public int getTelefone() {
        return telefone;
    }

    public void setTelefone(int telefone) {
        this.telefone = telefone;
    }

    public int getIdentidade() {
        return identidade;
    }

    public void setIdentidade(int identidade) {
        this.identidade = identidade;
    }

}

File: 2° Periodo/Prog 2  JAVA C++/LAB 10/src/Pessoa.cpp
#include "Pessoa.hpp"
#include "ExcecaoSexoInvalido.hpp"
#include <iostream>

Pessoa::Pessoa() {}

Pessoa::Pessoa(string nome,char sexo, string endereco,int cpf,string telefone,string identidade){
    this->nome = nome;
    this->endereco=endereco;
    this->cpf = cpf;
    this->telefone = telefone;
    this->identidade = identidade;
    this->sexo = sexo;
}
// Getters
string Pessoa::getNome() {
    return nome;
}

char Pessoa::getSexo() {
    return sexo;
}

string Pessoa::getEndereco() {
    return endereco;
}

int Pessoa::getCpf() {
    return cpf;
}

string Pessoa::getTelefone() {
    return telefone;
}

string Pessoa::getIdentidade() {
    return identidade;
}

// Setters
void Pessoa::setNome(string novoNome) {
    nome = novoNome;
}

void Pessoa::setSexo(char novoSexo) {
    if(novoSexo != 'M' && novoSexo != 'F'){
        throw ExcecaoSexoInvalido();
        return;
    }
    sexo = novoSexo;
}

void Pessoa::setEndereco(string novoEndereco) {
    endereco = novoEndereco;
}

void Pessoa::setCpf(int novoCpf) {
    cpf = novoCpf;
}

void Pessoa::setTelefone(string novoTelefone) {
    telefone = novoTelefone;
}

void Pessoa::setIdentidade(string novaIdentidade) {
    identidade = novaIdentidade;
}

void Pessoa::imprimir() {
    cout << "Nome: " << nome << endl;
    cout << "Sexo: " << sexo << endl;
    cout << "Endereco: " << endereco << endl;
    cout << "CPF: " << cpf << endl;
    cout << "Telefone: " << telefone << endl;
    cout << "Identidade: " << identidade << endl;
}
